src/process.cr

```diff
@@ -273,6 +273,16 @@ class Process
     @waitpid.closed? || !exists_system?
   end
 
+  # Terminate process gracefully
```

Comment:
Does this exist on windows?

Reply:
Yes.

Reply:
How? I can't see a way to terminate an arbitrary process gracefully. Go doesn't support it.

Reply:
Most of time you deal with GUI process, because Windows is about, surprise, windows ;). So you must find main window of target process and send WM_CLOSE message to it. If window not found, you can send WM_QUIT message to main thread. For console process, you should send crtl+c event to it using GenerateConsoleCtrlEvent.

Reply:
Go doesn't seem to want to deal with that complexity for graceful process termination. I guess we'll see how much code there is :)